web/portal/client/src/components/ActiveCalls/useRealtimeCalls.tsx

```diff
@@ -98,6 +98,7 @@ const useRealtimeCalls = (): [boolean, Calls] => {
     }
 
     console.log('Ignore update for unknown call', data);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
We can disable this rule if you need it in eslintrc.json.

Reply:
I've been able to remove this comment without whitelisting it